util.mli

```diff
@@ -68,13 +68,15 @@ exception Undefined of string * json
       return undefined. Currently this only happens when an array index is out
       of bounds. *)
 
-val ( |> ) : 'a -> ('a -> 'b) -> 'b
-  (** Forward pipe operator; useful for composing JSON access functions
-      without too many parentheses *)
```

Comment:
Removing it may break compatibility for some people, e.g. `let (|>) = Yojson.Util.(|>)`. We can instead mark it as obsolete, and define the value as `let (|>) = (|>)` in the ml file. Otherwise the rest looks good to me. Thanks!